server/app/test/v2/users/controller.spec.js

```diff
@@ -0,0 +1,93 @@
+import { expect } from 'chai';
+import request from 'supertest-as-promised';
+import app from '../../../index';
+import '../../util';
+
+describe('User routes', function userRoutes() {
+  describe('#signup', function signUp() {
+    const signup = '/api/v1/users';
+    const user = {
+      email: 'test@test.com',
+      password: 'password',
+      firstName: 'John',
+      lastName: 'Doe'
+    };
+
+    it('should send a token with valid user', function createUser() {
```

Comment:
User the affirmative when you can. `it('sends a token with valid user')`

http://betterspecs.org/ -- some of it is Ruby specific, but overall Ruby practitioners have set the standard for testing.